libs/dh/market-participant/actors/feature-actors/src/lib/create/steps/dh-new-organization-step.component.ts

```diff
@@ -80,63 +78,84 @@ import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
-          [options]="countryOptions"
-          [formControl]="newOrganizationForm().controls.country"
-        />
-        <watt-text-field
-          [formControl]="newOrganizationForm().controls.cvrNumber"
-          [label]="t('cvrNumber')"
-        >
-          @if (newOrganizationForm().controls.cvrNumber.hasError('invalidCvrNumber')) {
-            <watt-field-error>
-              {{ t('cvrInvalid') }}
-            </watt-field-error>
-          }
-        </watt-text-field>
-      </vater-stack>
-
-      <div class="column">
-        @if (this.isCvrBusy$() | push) {
+    <vater-stack gap="m" align="center" direction="row">
+      <watt-dropdown
+        translateKey="marketParticipant.actor.create.counties"
+        dhDropdownTranslator
+        [label]="t('country')"
+        [showResetOption]="false"
+        [options]="countryOptions"
+        [formControl]="newOrganizationForm().controls.country"
+      />
+      <watt-text-field
+        [formControl]="newOrganizationForm().controls.cvrNumber"
+        [label]="t('cvrNumber')"
+      >
+        @if (this.lookingForCVR()) {
           <watt-spinner [diameter]="22" />
         }
-      </div>
-    </div>
+        @if (newOrganizationForm().controls.cvrNumber.hasError('invalidCvrNumber')) {
+          <watt-field-error>
+            {{ t('cvrInvalid') }}
+          </watt-field-error>
+        }
+      </watt-text-field>
+    </vater-stack>
 
     <vater-stack gap="m" align="flex-start">
       <watt-text-field
         [formControl]="newOrganizationForm().controls.companyName"
         [label]="t('companyName')"
       />
-      <watt-text-field
-        [prefix]="'alternateEmail'"
-        [formControl]="newOrganizationForm().controls.domain"
-        [label]="t('domain')"
-      >
-        @if (newOrganizationForm().controls.domain.hasError('pattern')) {
-          <watt-field-error>
-            {{ t('domainInvalid') }}
-          </watt-field-error>
-        }
-      </watt-text-field>
+      @if (multipleDomainSupport) {
+        <vater-stack direction="row" gap="m" fill="horizontal">
+          <watt-text-field
+            [prefix]="'alternateEmail'"
+            [formControl]="newOrganizationForm().controls.domain"
+            [label]="t('domain')"
+          >
+            @if (newOrganizationForm().controls.domain.hasError('pattern')) {
+              <watt-field-error>
+                {{ t('domainInvalid') }}
+              </watt-field-error>
+            }
+          </watt-text-field>
+          <watt-button variant="text" (click)="addDomain()">{{ t('add') }}</watt-button>
+        </vater-stack>
+        <vater-flex wrap="wrap" direction="row" grow="0" gap="s" justify="flex-start">
+          @for (domain of newOrganizationForm().controls.domains.value; track domain) {
+            <watt-action-chip icon="remove" (click)="removeDomain(domain)">{{
+              domain
+            }}</watt-action-chip>
+          }
+        </vater-flex>
```

Comment:
issue: after adding the needed domains, one cannot continue to next step.

![image](https://github.com/user-attachments/assets/6d5f1542-6687-40d7-8d10-512a356d1f75)